README.md

```diff
@@ -42,7 +42,6 @@ AUR: [dualsensectl-git](https://aur.archlinux.org/packages/dualsensectl-git/)
 * meson
 * libhidapi-hidraw
 * libdbus-1
-* libudev
```

Comment:
Was this meant to be `libdbus-1`? `libudev` is still used in the code, while the dbus was removed everywhere.